src/lsst/cm/tools/db/sqlalch_interface.py

```diff
@@ -586,6 +586,26 @@ def load_error_types(self, config_yaml: str) -> None:
         error_code_dict = config_data["pandaErrorCode"]
         for key, val in error_code_dict.items():
             for error_name, error_type in val.items():
+                matched_id = conn.query(ErrorType.id).filter_by(error_name=error_name).first()
+                if matched_id is not None:
+                    stmt = (
+                        update(ErrorType)
+                        .where(ErrorType.id == matched_id[0])
```

Comment:
Do you need/want `[0]` here, given `first()` above?

Reply:
yes I do

Reply:
It don't work otherwise

Reply:
Well alrighty then!  It doesn't do what I thought it did, I guess -- will have to educate myself later.